artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -16,40 +16,6 @@
  */
 package org.apache.activemq.artemis.core.server.impl;
 
-import javax.management.MBeanServer;
```

Comment:
can you avoid these lines on the diff?

it gets difficult to track changes over time.

I don't think you really needed these.

Reply:
Will do.